src/SFA.DAS.LevyTransferMatching.Web/Controllers/PledgesController.cs

```diff
@@ -232,10 +233,51 @@ private void AddLocationErrorsToModelState(Dictionary<int, string> errors)
         public async Task<IActionResult> Applications(ApplicationsRequest request)
         {
             var response = await _orchestrator.GetApplications(request);
-
             return View(response);
         }
 
+        [Authorize(Policy = PolicyNames.ManageAccount)]
+        [HttpPost]
+        [Route("{encodedPledgeId}/applications")]
+        public IActionResult Applications(ApplicationsPostRequest request)
+        {
+            return RedirectToAction(nameof(RejectApplications), new { request.EncodedAccountId, request.EncodedPledgeId, request.ApplicationsToReject });
+        }
+
+        
+        [Authorize]
+        [Route("{encodedPledgeId}/applications/reject-applications")]
+        public async Task<IActionResult> RejectApplications(ApplicationsPostRequest request)
```

Comment:
While the model here might be identical to that used elsewhere, we should create a new model - `RejectApplicationsRequest` to represent the request to this action.

Reply:
yes sure, I have created a new model for RejectApplicationsRequest